### PR TITLE
Platform css

### PR DESCRIPTION
**Pull request by mikedao**
* Improved layout of Admin/Borrowers Index.

### CODE REVIEW

File: app/views/admin/tenants/index.html.erb [hunk 1]
```diff
@@ -1,69 +1,81 @@
...
+              <%= tenant.location %>
+            </td>
+            <td>
+              <%= button_to "Edit", edit_admin_tenant_path(tenant.id), class: 'btn btn-info btn-xs btn-block', method: :get %>
+            </td>
+          </tr>
+        <% end %>
       <% end %>
-    <% else %>
-      None
```
Comment (EmilyMB) [inline app/views/admin/tenants/index.html.erb]: We don't want none anymore? It seems like it might be OK to leave it in there.

Comment (mikedao) [inline app/views/admin/tenants/index.html.erb]: The way it is structured right now, it is no longer necessary. I think its better without.

File: config/routes.rb [hunk 2]
```diff
@@ -5,8 +5,8 @@
   get "/logout", to: "sessions#destroy", as: "logout"
 
   resources :tenants, only: [:new, :create, :index]
-  resources :categories, only: [:show, :index]
   resources :projects
+  resources :categories, only: [:index]
 
   get "/choose", to: "static_pages#choose"
```
Comment (EmilyMB) [inline config/routes.rb]: We don't even have this any more, do we?

**Comment by oorion** [comment]
looks great!

File: config/routes.rb (hunk 2 above)
Comment (mikedao) [inline config/routes.rb]: I thought you wanted this?

Comment (EmilyMB) [inline config/routes.rb]: Sorry, I meant the get "/choose"

Comment (mikedao) [inline config/routes.rb]: FIXED!

**Comment by EmilyMB** [comment]
:heart: that everything is lined up now :smile:

**Pull request merged**